### ISSUE TITLE
bug: all models fail to load on fresh windows installation on non-admin user account

### ISSUE THREAD

**Issue by Andydna2** [janhq/jan#2508]
I have installed Jan :Version: 0.4.92024-03-26T18:31:00.549Z [NITRO]::CPU informations - 9

**Steps to reproduce**

on Windows 11 (PL language version)  on user account without admin privilages 
16GB RAM, Ryzen 7735HS procesor with newest Adrenaline drivers for Radeon 680M

the instalation was smooth, but any models fail to load.
I tried manually importing some, but also to no avail

**Expected behavior**
just works :-)

2024-03-26T18:59:32.074Z [NITRO]::Debug: Nitro is ready
2024-03-26T18:59:32.074Z [NITRO]::Debug: Loading model with params {"ctx_len":4096,"prompt_template":"<|im_start|>system\n{system_message}<|im_end|>\n<|im_start|>user\n{prompt}<|im_end|>\n<|im_start|>assistant","llama_model_path":"C:\\Users\\Rafał\\jan\\models\\trinity-v1.2-7b\\trinity-v1.2.Q4_K_M.gguf","system_prompt":"<|im_start|>system\n","user_prompt":"<|im_end|>\n<|im_start|>user\n","ai_prompt":"<|im_end|>\n<|im_start|>assistant","cpu_threads":9,"ngl":100}
2024-03-26T18:59:32.102Z [NITRO]::Error: llama_model_load: error loading model: failed to open C:\Users\Rafał\jan\models\trinity-v1.2-7b\trinity-v1.2.Q4_K_M.gguf: No such file or directory
llama_load_model_from_file: failed to load model
llama_init_from_gpt_params: error: failed to load model 'C:\Users\Rafał\jan\models\trinity-v1.2-7b\trinity-v1.2.Q4_K_M.gguf'

2024-03-26T18:59:32.109Z [NITRO]::Debug: 94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m\[93m_[94m_[93m\[94m/[93m [94m [93m [94m [93m [94m [93m [94m [93m [94m\[93m_[94m_[93m\[94m/[93m [94m [93m [94m [0m
[0m20240326 18:59:31.809000 UTC 2864 INFO  Nitro version:  - main.cc:50
20240326 18:59:31.809000 UTC 2864 INFO  Server started, listening at: 127.0.0.1:3928 - main.cc:54
20240326 18:59:31.809000 UTC 2864 INFO  Please load your model - main.cc:55
20240326 18:59:31.809000 UTC 2864 INFO  Number of thread is:16 - main.cc:62
{"timestamp":1711479572,"level":"INFO","function":"loadModelImpl","line":642,"message":"system info","n_threads":9,"total_threads":16,"system_info":"AVX = 1 | AVX_VNNI = 0 | AVX2 = 1 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 1 | NEON = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 0 | SSE3 = 1 | SSSE3 = 0 | VSX = 0 | MATMUL_INT8 = 0 | "}
{"timestamp":1711479572,"level":"ERROR","function":"load_model","line":563,"message":"unable to load model","model":"C:\\Users\\Rafał\\jan\\models\\trinity-v1.2-7b\\trinity-v1.2.Q4_K_M.gguf"}
20240326 18:59:32.096000 UTC 3424 ERROR Error loading the model - llamaCPP.cc:646

2024-03-26T18:59:32.108Z [NITRO]::Debug: Load model success with response {}
2024-03-26T18:59:32.111Z [NITRO]::Debug: Validate model state with response 200
2024-03-26T18:59:32.113Z [NITRO]::Debug: Validate model state failed with response "OK"
2024-03-26T18:59:32.114Z [NITRO]::Error: Validate model status failed
2024-03-26T18:59:54.519Z [NITRO]::CPU informations - 9
2024-03-26T18:59:54.520Z [NITRO]::Debug: Request to kill Nitro
2024-03-26T18:59:57.619Z [NITRO]::Debug: Nitro process is terminated
2024-03-26T18:59:57.627Z [NITRO]::Debug: Nitro exited with code: 3221226505
2024-03-26T18:59:58.128Z [NITRO]::Debug: Spawning Nitro subprocess...
2024-03-26T18:59:58.129Z [NITRO]::Debug: Spawn nitro at path: C:\Users\Rafał\jan\extensions\@janhq\inference-nitro-extension\dist\bin\win-cpu\nitro.exe, and args: 1,127.0.0.1,3928
2024-03-26T18:59:58.164Z [NITRO]::Debug: [93m [94m [93m [94m [93m [94m [93m_[94m_[93m_[94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m [93m [94m

**Comment by vedantm8** [comment]
I am seeing the same problem on my end with Windows 11. 

I installed 2 models: TinyLlama Chat 1.1B Q4 and Wizard Coder Python 13B. 

While asking questions with TinyLlama model worked, asking questions utilizing the Wizard Coder model never returned an answer as the model would not load, seemingly stuck on "starting model wizardcoder-13b"

**Comment by Van-QA** [comment]
Related to https://github.com/janhq/jan/issues/2489. @hiro-v

**Comment by Van-QA** [comment]
@Andydna2, Another issue from your log, is that the folder path is: Rafał => Please try this workaround solution for the time being: https://github.com/janhq/jan/issues/1713#issuecomment-1912330258

Many thanks

**Comment by Andydna2** [comment]
> @Andydna2, Another issue from your log, is that the folder path is: Rafał => Please try this workaround solution for the time being: [#1713 (comment)](https://github.com/janhq/jan/issues/1713#issuecomment-1912330258)
yes, that helped - Thanks!
But maybe I have more than one problem
Because it only works sometimes, e.g. model load sometimes, and sometimes fails to load, especially after switching off  Vulcan:
```

2024-04-01T20:37:24.125Z [NITRO]::Debug: [1712003844] [D:\a\nitro\nitro\controllers\llamaCPP.h:  475][llama_client_slot::print_timings] 
[1712003844] [D:\a\nitro\nitro\controllers\llamaCPP.h:  480][llama_client_slot::print_timings] print_timings: prompt eval time =    4766.57 ms /    83 tokens (   57.43 ms per token,    17.41 tokens per second)
[1712003844] [D:\a\nitro\nitro\controllers\llamaCPP.h:  485][llama_client_slot::print_timings] print_timings:        eval time =   32021.02 ms /   246 runs   (  130.17 ms per token,     7.68 tokens per second)
[1712003844] [D:\a\nitro\nitro\controllers\llamaCPP.h:  487][llama_client_slot::print_timings] print_timings:       total time =   36787.59 ms
[1712003844] [D:\a\nitro\nitro\controllers\llamaCPP.h: 1585][llama_server_context::update_slots] slot 0 released (330 tokens in cache)

2024-04-01T20:38:31.232Z [NITRO]::Debug: Request to kill Nitro
2024-04-01T20:38:31.517Z [NITRO]::Debug: 20240401 20:37:24.126000 UTC 25056 INFO  reached result stop - llamaCPP.cc:365
20240401 20:37:24.126000 UTC 25056 INFO  End of result - llamaCPP.cc:338
20240401 20:37:24.203000 UTC 25212 INFO  Task completed, release it - llamaCPP.cc:408
20240401 20:38:31.517000 UTC 24996 INFO  Program is exitting, goodbye! - processManager.cc:8
20240401 20:38:31.517000 UTC 24996 INFO  changed to false - llamaCPP.cc:680
[1712003911] [D:\a\nitro\nitro\controllers\llamaCPP.h: 1585][llama_server_context::update_slots] slot 0 released (330 tokens in cache)

2024-04-01T20:38:35.899Z [NITRO]::Debug: Nitro process is terminated
2024-04-01T20:38:35.901Z [NITRO]::Debug: Nitro exited with code: 3221226505
2024-04-01T20:39:05.352Z [NITRO]::CPU informations - 9
2024-04-01T20:39:05.353Z [NITRO]::Debug: Request to kill Nitro
2024-04-01T20:39:05.357Z [NITRO]::Debug: Nitro process is terminated
2024-04-01T20:39:05.860Z [NITRO]::Debug: Spawn nitro at path: C:\Users\vojevoda\jan\extensions\@janhq\inference-nitro-extension\dist\bin\win-cuda-12-0\nitro.exe, and args: 1,127.0.0.1,3928
2024-04-01T20:39:05.859Z [NITRO]::Debug: Spawning Nitro subprocess...
2024-04-01T20:39:08.154Z [NITRO]::Debug: Nitro exited with code: 3221225781
2024-04-01T20:39:08.155Z [NITRO]::Error: child process exited with code 3221225781
2024-04-01T20:39:29.180Z [NITRO]::CPU informations - 9
```
So it seems Nitro make a wrong selection - I do not have Cuda on my laptop. That seems like an error, that is unless the default of `win-cuda-12-0` is to infer on CPU when no GPU is present ?




> Hi @vedantm8 Could you share the CPU/ GPU/ RAM you have so that I can help you? What I can tell now is that you have small RAM/ GPU VRAM (4GB/ 6GB) so that you can only use TinyLlama. `wizardcoder-13b` might need around 8-12GB VRAM to use.

As I a wrote in the original issue I test Jan on a laptop with iGPU , so I can play with VRAM alocation - but inference fails to run on modern AMD CPU.
>16GB RAM, Ryzen 7735HS procesor with newest Adrenaline drivers for Radeon 680M

On a final topic, I did however have some limited luck with Experimental Vulcan - and since this topic interest me I would be happy to get involoved in my (limited) free time to move it along. For instance it seems that inference performance kicks-in  with more signifciant speed up under Linux compared to Win11.
Is there anything specific that is a pain but the same time isolated and easy enough for fresh eye to get involed in - either at Nitro or llama level ?

**Comment by Van-QA** [comment]
hi @Andydna2, 
While the Jan app is using GPU, can you give us the settings.json?

**Comment by Andydna2** [comment]
Fails to run on CPU:
```
  "notify": true,
  "run_mode": "gpu",
  "nvidia_driver": {
    "exist": false,
    "version": ""
  },
  "cuda": {
    "exist": false,
    "version": ""
  },
  "gpus": [],
  "gpu_highest_vram": "",
  "gpus_in_use": [
    ""
  ],
  "is_initial": false,
  "vulkan": false
```
And this setting when it runs ok on iGPU as Vulcan: - but with somewhat slow performance of under 7 t/s (Radeon 680M)
```{
  "notify": true,
  "run_mode": "gpu",
  "nvidia_driver": {
    "exist": false,
    "version": ""
  },
  "cuda": {
    "exist": false,
    "version": ""
  },
  "gpus": [
    {
      "id": "0",
      "vram": "0",
      "name": "AMD Radeon(TM) 680M",
      "arch": "unknown"
    }
  ],
  "gpu_highest_vram": "",
  "gpus_in_use": [
    "0"
  ],
  "is_initial": false,
  "vulkan": true
}```


When I manually edit JSON to modify `run_mode": "cpu",`  --> **it runs!**  so this is a fairly simple workaorund for me. Thanks.
However the inference aswer speed is fairly similar to Vulcan performance at about 7 t/s.
Does any one know if indeed Ryzen 7735HS is that fast , or 680M is that slow that they are on-par  ?

**Comment by Van-QA** [comment]
Tested and the issue was resolved as of Jan v0.4.10-360 ✅

**Comment by Andydna2** [comment]
@Van-QA thanks! I can confirm that `10-360` fixes the issue. Now inference works after toggling on/off and also works out-of-the-box on entirely fresh Jan installation
